package.json

```diff
@@ -68,6 +68,7 @@
     "@babel/eslint-parser": "^7.19.1",
     "@folio/eslint-config-stripes": "^6.1.0",
     "@folio/stripes": "^7.0.0 || ^8.0.0",
+    "@folio/stripes-acq-components": "^3.0.0 || ~4.0.0",
```

Comment:
Why `~` rather than the usual `^`? The whole promise of semantic versioning is that `^` will never get a version with breaking changes.

Reply:
we faced with it 1 or 2 years ago, it's not about breaking changes actually

e.g. there are 2 releases:
Release A and Release B
having ^ dependency for shared library caused a case that UI build for Release A hotfix picked Release B version when Release B used new stripes functionality. At the same time stripes version is strictly defined in release branches (https://github.com/folio-org/platform-complete/blob/master/package.json#L70)

Reply:
Hang on, are you saying that a minor-version-compatible release of stripes-acq-components was dependent on a newer version of Stripes? But that shouldn't be a problem so long as we're also using `^` for the version of Stripes itself. Right?

Reply:
all our modules depend on stripes, I'll extend my example with numbers

Release A
stripes: 1.0.0
stripes-acq-components: 1.0.0
module-a: 1.0.0 (has stripes-acq-components: ^1.0.0 dependency)

Release B
stripes: 1.1.0
stripes-acq-components: 1.1.0
module-a: 1.1.0 (has stripes-acq-components: ^1.0.0 dependency)

Release A hotfix (was approved when Release B releases are done)
stripes: 1.0.0
module-a: 1.0.0 (has stripes-acq-components: ^1.0.0 dependency)
stripes-acq-components: 1.1.0

and here we have 1.1.0 because in module we used ^1.0.0, 1.1.0 doesn't contain any breaking changes, it just uses new functionality from stripes, but this functionality is not available because stripes version is strictly defined, in hotfix it's still 1.0.0

Reply:
@MikeTaylor 

in our case it's case with `it's inherently the case that any version of module-a running under it is constrained to Stripes 1.0.0` as for final releases exact version is used without any `^` or `~`

and even if we define `stripes: ^1.1.0` in `stripes-acq-component@1.1.0` to tell exactly that we rely on new version and functionality, we'' be in situation when in `platform-complete` we have `stripes: 1.0.0`, but `stripes-acq-components` requires 1.1.0 just because app modules define dependency as `stripes-acq-components: ^1.0.0`

that's why we want to exclude any possibility to install new minor version for previous releases - currently it's done via resolution in platform-complete, and we'd like to remove it

Reply:
OK, thanks for going over all that.